src/sentry/api/endpoints/release_threshold.py

```diff
@@ -64,3 +72,30 @@ def post(self, request: Request, project: Project) -> HttpResponse:
             environment=result.get("environment"),
         )
         return Response(serialize(release_threshold, request.user), status=201)
+
+    def get(self, request: Request, project: Project) -> HttpResponse:
+        request.data["project"] = project.slug
+        serializer = ReleaseThresholdGETSerializer(
+            data=request.data,
+            context={
+                "organization": project.organization,
+                "access": request.access,
+            },
+        )
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+        environment_name: Union[str, None] = request.GET.get("environment")
+        if not environment_name:
+            release_thresholds = ReleaseThreshold.objects.filter(project=project)
+            return Response(serialize(list(release_thresholds), request.user), status=200)
```

Comment:
rather than having separate queries + responses, can we just ocnstruct the query and have a single return at the bottom?

```
q = Query({
    project=project
})
if environment_name:
    try:
        environment = ...
    ...
    q.environment = environment

release_thresholds = ReleaseThresholds.object.filter(q)
return Response(serialize(list(release_thresholds)))
```